src/crypto/sign.js

```diff
@@ -89,7 +89,7 @@ export function verifyMessageWithPublicKey(signedMessage, publicKey) {
 	if (signatureVerified) {
 		return naclInstance.decode_utf8(signatureVerified);
 	}
-	throw new Error('Invalid signature publicKey combination, cannot verify message');
+	return false;
```

Comment:
Yuidoc return types. Do we want to always return a boolean instead of sometimes a boolean, sometimes a string?